Scatt.java

```diff
@@ -27,13 +27,13 @@
      */
     public static void main(String[] args)
     {
-        System.out.println("Please enter the folder name: ");
+        System.out.print("Please enter the folder path: ");
         String dirName = System.console().readLine();
```

Comment:
@haietza Sure!  I mean really I think it's fine either way, just tripped me up for a sec!